partcad/src/partcad/lint/all.py

```
@@ -0,0 +1,29 @@
+
+PARTCAD_SCHEMA = None
+_global_lint_checks = []
+
+def get_partcad_schema():
+    global PARTCAD_SCHEMA
+    if PARTCAD_SCHEMA is None:
+        with importlib.resources.files("partcad.schema").joinpath("partcad.json").open("r") as file:
+            PARTCAD_SCHEMA = json.load(file)
+    return PARTCAD_SCHEMA
+
+def get_linting_checks(concurrency_cap: int) -> list[Linting]:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)